R/adverb-safely.R

```diff
@@ -10,7 +10,11 @@
 #' function (a verb). If you'd like to include a function created an adverb
 #' in a package, be sure to read [faq-adverbs-export].
 #'
-#' @inheritParams map
+#' @param .f A function to modify, specified in one of the following ways:
+#'   * A named function, e.g. `mean`.
+#'   * An anonymous function, e.g. `function(x) x + 1`.
+#'   * A lambda function, e.g. `\(a) a + 1`.
```

Comment:
Just so that we don't imply that the shorthand variant needs different naming conventions:

```suggestion
#'   * A lambda function, e.g. `\(x) x + 1`.
```